packages/odyssey-react-mui/src/themes/odyssey/typography.ts

```diff
@@ -0,0 +1,109 @@
+    fontWeightMedium: "500",
+    fontWeightBold: "600",
+    h1: {
+      fontFamily:
+        "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
+      fontWeight: "600",
+      fontSize: "2.571rem",
+      lineHeight: "1.25",
+      letterSpacing: "initial",
+    },
+    h2: {
+      fontFamily:
+        "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
+      fontWeight: "600",
+      fontSize: "2.286rem",
+      lineHeight: "1.28571429",
+      letterSpacing: "initial",
+    },
+    h3: {
+      fontFamily:
+        "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
+      fontWeight: "600",
+      fontSize: "2rem",
+      lineHeight: "1.28",
+      letterSpacing: "initial",
+    },
+    h4: {
+      fontFamily:
+        "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
+      fontWeight: "600",
+      fontSize: "1.571rem",
+      lineHeight: "1.27272727",
+      letterSpacing: "initial",
+    },
+    h5: {
+      fontFamily:
+        "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
+      fontWeight: "600",
+      fontSize: "1.286rem",
+      lineHeight: "1.333333333",
+      letterSpacing: "initial",
+    },
+    h6: {
+      fontFamily:
+        "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
+      fontWeight: "600",
+      fontSize: "1.143rem",
+      lineHeight: "1.25",
+      letterSpacing: "initial",
+    },
+    subtitle1: undefined,
+    subtitle2: undefined,
+    body1: {
+      fontFamily:
+        "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
+      fontWeight: "400",
+      fontSize: "1rem",
+      lineHeight: "1.42857143",
+      letterSpacing: "initial",
+    },
+    body2: undefined,
+    button: undefined,
+    caption: {
+      fontFamily:
+        "'Public Sans', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', 'Noto Sans Arabic', sans-serif",
+      fontWeight: "400",
+      fontSize: "0.875rem",
+      lineHeight: "1.42857143",
+      letterSpacing: "initial",
+    },
+    overline: undefined,
+  },
+});
+
+// Update the Typography's variant prop options
+// Unsure where we want to store this
+declare module "@mui/material/Typography" {
```

Comment:
@jeffbelser-okta where should these associated TS declarations to live?

Reply:
I've disabled a few of the existing variants in the theme above, since we won't be offering them. Following this guidance (https://mui.com/material-ui/customization/typography/#adding-amp-disabling-variants) we also need to update their typing so that the `<Typography>` component doesn't accept them.

Reply:
Will remove for this PR. TODO: set `undefined` styles to Body1